### DISCUSSION TITLE
C:/URBANopt-cli-0.9.0/gems/ruby/2.7.0/gems/urbanopt-geojson-0.9.0/lib/urbanopt/geojson/feature.rb:277:in `block in find_feature_center': undefined method `/' for #<Array:0x000000000aec6af8> (NoMethodError)

### DISCUSSION THREAD

**Discussion by muel1994** [urbanopt/urbanopt-cli#408] [Q&A]
![image](https://user-images.githubusercontent.com/92198662/215015052-13664860-451e-4085-990a-0e0af1283183.png)

uo run -f "~~~.geojson" -s "~~~honeybee_scenario.csv"
command was executed through cmd.

And the following error occurred.
C:/URBANopt-cli-0.9.0/gems/ruby/2.7.0/gems/urbanopt-geojson-0.9.0/lib/urbanopt/geojson/feature.rb:277:in `block in find_feature_center': undefined method `/' for #<Array:0x000000000aec6af8> (NoMethodError)

What should I do to fix this error? Any help please?

**Comment by tanushree04** [comment]
Please use the latest URBANopt installer (http://urbanopt-cli-installers.s3-website-us-west-2.amazonaws.com/) and reference the URBANopt documentation (https://docs.urbanopt.net/) for steps on how to run URBANopt. Feel free to reach out if you are still facing any issues!
